File: Patterns/Facade Pattern/CSharp/HomeTheater/Screen.cs
using System;

namespace HomeTheater
{
    public class Screen
    {
        private string description;

        public Screen(string description)
        {
            this.description = description;
        }

        public void Up() => Console.WriteLine($"{description} going up");

        public void Down() => Console.WriteLine($"{description} going down");

        public string Description { get => description; private set => description = value; }
    }
}

File: Patterns/Facade Pattern/Cpp/HomeTheater/Screen.cpp
#pragma once

#include <iostream>
#include <string>

using namespace std;

class Screen {
private:
    string description;

public:
    Screen(string description) : description{description} {}

    void up() {
        cout << description << " going up\n";
    }

    void down() {
        cout << description << " going down\n";
    }

    string toString() {
        return description;
    }
};